libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3916,6 +3916,9 @@ void GCS_MAVLINK::handle_common_message(const mavlink_message_t &msg)
     case MAVLINK_MSG_ID_GIMBAL_MANAGER_SET_ATTITUDE:
         handle_mount_message(msg);
         break;
+    case MAVLINK_MSG_ID_GIMBAL_MANAGER_SET_PITCHYAW:
```

Comment:
We can save a little flash by just putting the two cases one before the other like below:

```
case MAVLINK_MSG_ID_GIMBAL_MANAGER_SET_ATTITUDE:
case MAVLINK_MSG_ID_GIMBAL_MANAGER_SET_PITCHYAW:
        handle_mount_message(msg);
        break;
```